From d77542338dc909f72587411cb2ab8b08747d4629 Mon Sep 17 00:00:00 2001
From: Jonathan Brielmaier <jbrielmaier@suse.de>
Date: Wed, 12 Sep 2018 10:01:17 +0200
Subject: [PATCH 1/1] libguile-ssh: Remove ssh_threads from LDFLAGS.

libssh 0.8 doesn't include libssh_threads.so anymore. So without this
commit, libguile-ssh fails to build.
---
 libguile-ssh/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/guile-ssh-0.11.3/libguile-ssh/Makefile.am b/guile-ssh-0.11.3/libguile-ssh/Makefile.am
index 1ffb6c0..aebae41 100644
--- a/guile-ssh-0.11.3/libguile-ssh/Makefile.am
+++ b/guile-ssh-0.11.3/libguile-ssh/Makefile.am
@@ -44,7 +44,7 @@ BUILT_SOURCES = auth.x channel-func.x channel-type.x error.x \
 
 libguile_ssh_la_CPPFLAGS = $(GUILE_CFLAGS)
 
-libguile_ssh_la_LDFLAGS = -module -no-undefined -lssh_threads -lssh \
+libguile_ssh_la_LDFLAGS = -module -no-undefined -lssh \
 	-version-info $(LIBGUILE_SSH_INTERFACE) $(GUILE_LDFLAGS)
 
 AM_CFLAGS = $(WARN_CFLAGS)
-- 
2.16.4

